tcms/testruns/static/testruns/js/new.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(() => {
+
+    $('.set-running-button').on('click', () => {
+        console.log('clicked set to running')
+        
+        // get test run id
+        
+        // use the JSON RPC API to update the status
+        
+        // update the UI
+    })
```

Comment:
Empty method should be removed before merging to the branch. Each piece that we merge should be relatively self contained and fully functional b/c we may decide to release a page version with missing functionality at any moment.